code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -608,6 +610,44 @@
 		M.visible_message("<span class='notice'>[M] looks a bit dazed.</span>")
 	return ..()
 
+/datum/reagent/medicine/diphenhydramine/overdose_process(mob/living/M, severity)
+	var/list/overdose_info = ..()
+	var/effect = overdose_info[REAGENT_OVERDOSE_EFFECT]
+	var/update_flags = overdose_info[REAGENT_OVERDOSE_FLAGS]
```

Comment:
We should frankly move away from this system, as it straight up jarringly forces effects to happen simultaneously and is clunky to work with. Check Stimulative Agent/Sanguine Reagent for how you can simply ignore it

Reply:
I agree with the notion that we should move away from this system, but I'm not sure about doing it right now? I feel like doing it piecemeal kind of bandaids over the problem which needs one big coherent push.

I'm probably going to leave it in-line with the rest of the code for now

Reply:
Agreed, to keep it consistent this would need a big overhaul all at once, so it can be better adjusted with one another, this is fine for now